www/havm10/SeminarPage02/css/main.css

```diff
@@ -0,0 +1,188 @@
+* {
+    box-sizing: border-box;
+}
+
+.MapPart {
+    display: flex;
+    width: 100%;
+}
+
+.content {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 100%;
+    margin: 10px;
+}
+
+.save-content-conteiner {
+    max-width: 25%;
+    padding-left: 5px;
+    padding-right: 20px;
+    min-width: 25%;
+    border-width: 2px; 
+    border-style: solid; 
+    border-radius: 10px;
+}
+
+.markers {
+    display: flex; 
+    flex-wrap: wrap;
+    width: 40%;
+    border-width: 2px; 
+    border-style: solid; 
+    margin-left: 10px;
+    margin-right: 10px;
+    border-radius: 10px;
+}
+
+.sections {
+    display: flex;
+    flex-wrap: wrap;
+    min-width: 100%;
+    margin: 10px;
+}
+
+.section {
+    min-width: 50%;
+}
+
+
+.output {
+    display: flex;
+    width: 100%;
+    width: 40%;
+    padding-left: 5px;
+    padding-right: 5px;
+    border-width: 2px; 
+    border-style: solid; 
+    border-radius: 10px;
+}
+
+.data{
+    min-width: 60%;
+}
+
+#canvas-draw {
+    padding: 10px;
+    height: 300px;
+    border: 1px;
+    border-style: solid;
+    width: 35%;
+    border-width: 2px; 
+    border-style: solid; 
+    border-radius: 10px;
+}
+
+.column {
+    margin: 10px;
+    background-color: blue;
+    color: white;
+    text-align: center;
+    border-radius: 10px;
+    font-size: 18px;
+    margin-left: 15%;
+    margin-right: 15%;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+button {
+    width: 100%;
+    height: 50px;
+    margin-left: 25%;
+    margin-right: 25%;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    border-radius: 10px;
+    font-size: 15px;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+button:hover {
+    background-color: green;
+    color: white;
+}
+
+input {
+    width: 50%;
+    text-align: center;
+    margin: 10px;
+    border-radius: 10px;
+}
+
+#save-route {
+    width: 50%;
+    margin: 10px;
+}
+
+.save-name {
+    width: 70%;
+    padding-left: 20px;
+    color: white;
+}
+
+ul {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 100%;
+}
+
+li {
+    display: flex;
+    width: 100%;
+    margin: 5px;
+    height: 60px;
+    line-height: 60px;
+    margin-bottom: 15px; 
+    background-color: blue;
+    border-radius: 10px;
+}
+
+.save-delete {
+    width: 30%;
+    margin: 0px;
+    text-align: center;
+    height: 40px;
+    margin: 10px;
+}
+
+#map {
+    width: 60%;
+    height: 500px;
+    touch-action: none;
+    border-radius: 15px;
+    margin-left: 10px; 
+    border-width: 2px; 
+    border-style: solid; 
+}
+
+#save{
+    display: flex;
+    width: 100%;
+}
+
+h1 {
+    font-size: 40px;
+    font-family: Arial, Helvetica, sans-serif;
```

Comment:
to font-family se opakuje, nestaci to dat jednou do body?

Reply:
přidáno do body